templates/download/index.twig

```diff
@@ -27,116 +24,105 @@
 </div>
 {% endmacro %}
 
-{% import _self as download %}
-
-{% import 'macros.twig' as macros %}
-
-{% block content %}
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
-{{ macros.jumbo('Download LMMS'|trans,
-'Downloading and using LMMS is free! Just choose the operating system you want to run LMMS on:'|trans) }}
-<div class="text-center container">
-		<div>
-		<div class="btn-group" data-toggle="buttons">
-			<label class="btn btn-default" onclick="showOS('#linux')">
-				<input type="radio" name="options" id="linux-button"><span class="fab fa-linux fa-3x"></span><br>Linux
-			</label>
-			<label class="btn btn-default" onclick="showOS('#windows')">
-				<input type="radio" name="options" id="windows-button"><span class="fab fa-windows fa-3x"></span><br>Windows
-			</label>
-			<label class="btn btn-default" onclick="showOS('#mac')">
-				<input type="radio" name="options" id="mac-button"><span class="fab fa-apple fa-3x"></span><br>macOS
-			</label>
-		</div>
-	</div>
-</div>
+{% macro osbutton(os, icon, title) %}
+<label class="btn btn-default" onclick="showOS('#{{ os }}')">
+	<input type="radio" name="options" id="{{ os }}-button"><span class="fab {{ icon }} fa-3x"></span><br>{{ title }}
+</label>
+{% endmacro %}
 
-<div id="linux-div" class="text-center hidden container">
-	<h2>{% trans %}Install LMMS on Linux{% endtrans %}</h2>
-	<p>{% trans %}Click one of the buttons below to download LMMS for Linux.{% endtrans %}</p>
-	{% if linstable %}
+{% macro ospage(os, title, subtitle, stable, pre, nightly, instruction, trailer) %}
+<div id="{{ os }}-div" class="text-center hidden container">
+	<h2>{{ title }}</h2>
+	<p>{{ subtitle }}</p>
+	{% if stable %}
 		<h3>{% trans %}Stable Versions{% endtrans %}</h3>
-		{% for lin in linstable %}
-			{{ download.printrelbutton(lin) }}
+		{% for asset in stable %}
+			{{ download.printrelbutton(asset) }}
 		{% endfor %}
-		{{ download.releasenotes(linstable[0]['release']['body'], 'linstable') }}
-		<p>{% trans %}Run <code>chmod +x ~/Downloads/*.AppImage</code> to set the <code>.AppImage</code> executable before running{% endtrans %}</p>
+		{{ download.releasenotes(stable[0].description, "#{os}-stable") }}
+		{% if instruction %}<p>{{ instruction|raw }}</p>{% endif %}
 	{% endif %}
-
-	{% if linpre %}
+	{% if pre %}
 		<h3>
+		{% if 'alpha' in pre[0].gitRef %}
 		{% if 'alpha' in linpre[0].release.tag_name %}
```

Comment:
This appears to be a typo. (`if`, `if`).

Reply:
I think I made a mistake when doing a branch update. Will fix it now.

Reply:
Done.